client/layout/masterbar/masterbar-cart/masterbar-cart-button.tsx

```diff
@@ -19,9 +19,10 @@ export type MasterbarCartButtonProps = {
 	onRemoveProduct?: ( uuid: string ) => void;
 	onRemoveCoupon?: () => void;
 	forceShow?: boolean;
+	showCount?: boolean;
 	checkoutLabel?: string;
-	cartIcon?: React.ElementType;
-	emptyCart?: React.ElementType;
+	cartIcon: React.ReactNode;
+	emptyCart: React.ReactNode;
```

Comment:
I think you still want these two props (`cartIcon` and `emptyCart`) to be optional, right?

Reply:
Thanks or spotting this. I have updated it to be optional